core/geometry/src/polyface/PolyfaceBuilder.ts

```diff
@@ -1934,17 +2010,21 @@ export class PolyfaceBuilder extends NullGeometryHandler {
       }
     }
   }
-
   /**
-   * * Create (and add to the builder) quad facets for a mitered pipe that follows a centerline curve.
-   * * Circular or elliptical pipe cross sections can be specified by supplying either a radius, a pair of semi-axis lengths, or a full Arc3d.
-   *    * For semi-axis length input, x corresponds to an ellipse local axis nominally situated parallel to the xy-plane.
-   *    * The center of Arc3d input is translated to the centerline start point to act as initial cross section.
+   * Create (and add to the builder) quad facets for a mitered pipe that follows a centerline curve.
+   * * Circular or elliptical pipe cross sections can be specified by supplying either a radius, a pair of semi-axis
+   * lengths, or a full Arc3d.
+   * * For semi-axis length input, x corresponds to an ellipse local axis nominally situated parallel to the xy-plane.
+   * * The center of Arc3d input is translated to the centerline start point to act as initial cross section.
```

Comment:
The extra space between the asterisks is necessary to create a nested list.

```suggestion
   * Add quad facets along a mitered pipe that follows a centerline curve.
   * * Circular or elliptical pipe cross sections can be specified by supplying either a radius, a pair of semi-axis
   * lengths, or a full Arc3d:
   *    * For semi-axis length input, x corresponds to an ellipse local axis nominally situated parallel to the xy-plane.
   *    * For Arc3d input, the center is translated to the centerline start point to act as initial cross section.
```